core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileDeleteAction.java

```diff
@@ -97,9 +97,10 @@ public boolean accept(Path path) {
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     if (config.fileRegex != null) {
       try {
-        Pattern filter = Pattern.compile(config.fileRegex);
+        Pattern.compile(config.fileRegex);
       } catch (Exception e) {
-        throw new IllegalArgumentException(String.format("File regex is invalid", e));
+        throw new IllegalArgumentException(String.format("File regex %s is invalid: %s",
+                                                         config.fileRegex, e.getMessage(), e));
```

Comment:
again... the exception is not an argument to String.format. It is an argument to the constructor.